test/expiry.test.ts

```diff
@@ -137,3 +137,51 @@ test('returns cookies before the max age', async () => {
     ['cookieName', { expires, name: 'cookieName', maxAge: 2, value: 'abc-123' }],
   ])
 })
+
+test('does overwrite exisiting cookies which would not expire', () => {
```

Comment:
Sorry for the nitpick, but I think this test name is quite hard to read...
If I understand the intention behind the test correctly, I may suggest paraphrasing it to:

```js
test('deletes an existing cookie when setting a cookie with the same name and expiration date in the past', () => {
```

Reply:
Yes, I agree. Your suggestion is much better.